modules/_labs/breadcrumbs/react/README.md

```diff
@@ -0,0 +1,271 @@
+# Canvas Kit React Breadcrumbs
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/_labs/README.md">
+  <img src="https://img.shields.io/badge/LABS-alpha-orange" alt="LABS: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+
+Breadcrumbs component for navigation through different levels of some structure.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-react-breadcrumbs
+```
+
+## Usage
+
+Breadcrumbs is a composable component that, while more verbose, offers more flexibility to build for
+your use case. There are two main variations: `List` and `CollapsibleList`. First we'll discuss
+`List`. There will also be information about the shared sub-components below.
+
+### `List`
+
+The `List` component is a styled, unordered list (`ul`) with no additional behaviors. Use this when
+you're not concerned about collapsing the list of breadcrumb items into a dropdown menu at a
+specified width.
+
+#### Component Props
+
+This component supports all native `li` props.
+
+#### Usage
+
+```tsx
+import React from 'react';
+import Breadcrumbs from '@workday/canvas-kit-labs-react-breadcrumbs';
+
+const AccountInvoiceBreadcrumbNav = () => {
+  return (
+    <Breadcrumbs.Nav aria-label="Breadcrumb">
+      <Breadcrumbs.List>
+        <Breadcrumbs.ListItem>
+          <Breadcrumbs.Link href="/account">Account</Breadcrumbs.Link>
+        </Breadcrumbs.ListItem>
+        <Breadcrumbs.ListItem>
+          <Breadcrumbs.Link href="/billing">Billing</Breadcrumbs.Link>
+        </Breadcrumbs.ListItem>
+        <Breadcrumbs.ListItem>
+          <Breadcrumbs.Link href="/invoices">Invoices</Breadcrumbs.Link>
+        </Breadcrumbs.ListItem>
+        <Breadcrumbs.CurrentItem>
+          2020_08_01_invoice.pdf
+        </Breadcrumbs.CurrentItem>
+      </BreadCrumbs.List>
+    </Breadcrumbs.Nav>
+  );
+}
```

Comment:
What was the reasoning behind having `Breadcrumbs.Link` in addition to `Breadcrumbs.ListItem` (as opposed to just `ListItem`)? Is there a situation (aside from `Breadcrumbs.CurrentItem`) where you would have an item with no `Link`?

Speaking of `CurrentItem`, I was curious about the need for a distinct `CurrentItem` component. Wouldn't the last `ListItem` always be the current item?

Or, are these both cases of us shifting towards more composable components (more verbose markup, but more flexibility as you mention in the README)? Am I meant to be able to have a `ListItem` with no `Link`, or a `List` with no `CurrentItem`?

Reply:
> Q: Why `ListItem` and `Link`?

Short answer: Composability
Long answer: I broke up the items semantically instead of one large component because they're doing different things. ListItem is mostly a styled `li`. It also adds a `>` separator after children are rendered and handles the truncation + tooltip functionality. The composable pattern also allows users to add anchor tag props directly to the `Link` instead of passing them through to the `ListItem`.

> Q: Why `CurrentItem` and not just another `ListItem`. You're correct that `CurrentItem` is mostly styled `li`, but it has some special behavior. It will never contain an `a`, so it renders a span, and there's no need for a `>` separator. It also needs to be smart about setting `tabindex` to `0` when the item is truncated. The other `ListItem`s don't have to think about that because their `Link` children act as the tab stops. I could have made the `Menu` smarter, mapped over the children, cloned the element + the needed props, but this felt simpler.